docs/conf.py

```diff
@@ -147,11 +148,12 @@ def __getattr__(cls, name):
 if on_rtd:
     nbsphinx_allow_errors = True
 else:
-    nbsphinx_allow_errors = False
+    nbsphinx_allow_errors = True
```

Comment:
I will set this back to `False`, I just wanted to see if everything else was fine.